src/db/dbhelpers.h

```diff
@@ -53,59 +53,59 @@ typedef std::function<void(const tntdb::Row&)> row_cb_f ;
 
 template <typename T>
 struct db_reply{
-    int status; // ok/fail
```

Comment:
It seems that the changes in `dbhelpers.h` are all about breaking indentation, and nothing functional. Can you pluck this one out of the PR please?